roles/nuage_master/handlers/main.yaml

```diff
@@ -0,0 +1,12 @@
+---
+- name: restart master
+  sudo: true
+  service: name=atomic-openshift-master state=restarted
+ 
+- name: restart nuagekubemon
+  sudo: true
+  service: name=nuagekubemon state=restarted
+
+- name: restart atomic-openshift-master
+  sudo: true
+  service: name=atomic-openshift-master state=restarted
```

Comment:
Looks like there are two handlers here to restart atomic-openshift-master...  One thing to keep in mind is that on an HA installation, you would most likely want to restart the api and/or the controllers service instead of the master service.